envs/Pendulum/Pendulum.py

```diff
@@ -10,11 +10,48 @@
 from gymnasium.envs.classic_control import utils
 from gymnasium.error import DependencyNotInstalled
 
+import tensorflow as tf
+from cmorl.utils.loss_composition import p_mean
+from cmorl.utils.reward_utils import CMORL, RewardFnType
+
 
 DEFAULT_X = np.pi
 DEFAULT_Y = 1.0
 
 
+def multi_dim_reward(state, action, env: "PendulumEnv"):
```

Comment:
You should be able to import PendulumEnv to use it as a type rather than the string "PendulumEnv"